src/components/grid-view/GridViewWrapper.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import * as React from 'react';
+import GridView from './GridView';
+
+type Props = {
+    currentCollection: Collection,
+    height: number,
+    slotRenderer: (slotIndex: number) => React.Element<any>,
+    width: number,
+};
+
+const DEFAULT_COLUMN_COUNT = 5;
+const MAX_COLUMN_COUNT = 5;
+
+const ONE_COLUMN_BREAKPOINT = 700;
+const THREE_COLUMN_BREAKPOINT = 1400;
+
+const GridViewWrapper = ({ width, ...rest }: Props) => {
+    let maxColumns;
```

Comment:
Does this number reflect `maxColumns` or just `columnCount`? Can the number of columns be changed in some other way, such as via CSS media queries or something?

Reply:
For now, `maxColumns` === `columnCount` and this is the only way the number of columns can be changed, but in the future, `columnCount` will be a prop passed in from `GridViewSlider`.  Does it make sense to keep this code that is anticipating the `GridViewSlider` or should I remove it?